src/hal/user_comps/wj200_vfd/wj200_vfd.comp

```diff
@@ -110,12 +116,30 @@ bool wj200_getStatus(modbus_t* ctx, wj200_status* status)
                 return false;
         }
 
+	// Read 1003h (output current) to 1019h (heatsink temp)
+        rc = modbus_read_registers(ctx, 0x1003-1, 1, currentRegister);
+
+        if(rc < 0)
+        {
+                return false;
```

Comment:
hm I notice that if the new reads fail for whatever reason, none of the output pins will be updated.  I don't know anything about wj200, but for instance if not all hardware versions support these registers then it seems this might break the behavior of the component when used with them.